index.bs

```diff
@@ -1450,6 +1451,62 @@ that are returned to the caller when a new credential is created, or a new asser
         [=extension identifier=] → [=client extension output=] entries produced by the extension's
         [=client extension processing=].
 
+    :   {{PublicKeyCredential/toJSON()}}
+    ::  This operation returns {{RegistrationResponseJSON}} or {{AuthenticationResponseJSON}},
+        which are [=JSON types=] mirroring {{PublicKeyCredential}}, suitable for submission to a
+        [=[RP]=] server as an `application/json` payload. The [=client=] is in charge of
+        [serializing values to JSON as usual](https://webidl.spec.whatwg.org/#idl-tojson-operation),
+        but MUST take additional steps to first encode any {{ArrayBuffer}} values to {{DOMString}} values
+        using [=base64url encoding=] to ensure compatibility with JSON syntax.
+        {{RegistrationResponseJSON/clientExtensionResults|RegistrationResponseJSON.clientExtensionResults}} or
+        {{AuthenticationResponseJSON/clientExtensionResults|AuthenticationResponseJSON.clientExtensionResults}}
+        members MUST be set to the output of {{PublicKeyCredential/getClientExtensionResults()}},
+        with any {{ArrayBuffer}} values encoded to {{DOMString}} values using
+        [=base64url encoding=]. This MAY include {{ArrayBuffer}} values from extensions registered
+        in the IANA "WebAuthn Extension Identifiers" registry [[!IANA-WebAuthn-Registries]] but not
+        defined in [[#sctn-extensions]].
```

Comment:
I think we also need the same instruction for `transports`? Then let's put both that and the above "The clientExtensionResult member ..." each in its own paragraph to make it easier to read.

```suggestion
        defined in [[#sctn-extensions]].

        The {{AuthenticatorAttestationResponseJSON/transports|AuthenticatorAttestationResponseJSON.transports}}
        member MUST be set to the output of {{AuthenticatorAttestationResponse/getTransports()}}.
```